kie-pmml-trusty/kie-pmml-models/pom.xml

```diff
@@ -17,9 +17,9 @@
     <module>kie-pmml-models-regression</module>
     <module>kie-pmml-models-drools</module>
     <module>kie-pmml-models-mining</module>
-    <module>kie-pmml-models-tests</module>
     <module>kie-pmml-models-tree</module>
     <module>kie-pmml-models-clustering</module>
+    <module>kie-pmml-models-tests</module>
```

Comment:
Out of curiosity. Is there any specific reason for moving tests module? I think that Maven should correctly find out the correct build order from modules dependencies.

Reply:
@jiripetrlik 
just for clarity - surely maven sort out the correct order, but I would like to keep those definitions "in good order" for easy of ready and, if ever needed, refactoring